diff --git a/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.js b/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.js
index 739ddec..19de797 100644
--- a/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.js
+++ b/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.js
@@ -2,13 +2,23 @@ import React from "react";
 import { TouchableOpacity } from "react-native";
 import styles from "./ImagedCardView.style";
 import CardContainer from "./components/CardContainer/CardContainer";
+import {View } from 'react-native';
 
 const ImagedCardView = props => {
   const { onPress } = props;
   return (
+    <View styles = {{shadowColor: "#000",
+        shadowOffset: {
+            width: 0,
+            height: 8,
+        },
+        shadowOpacity: 0.44,
+        shadowRadius: 10.32,
+
+        elevation: 16,}}> 
     <TouchableOpacity onPress={onPress} style={styles.container}>
       <CardContainer {...props} />
-    </TouchableOpacity>
+    </TouchableOpacity></View>
   );
 };
 
diff --git a/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.style.js b/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.style.js
index 3546dec..e6ce3f8 100644
--- a/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.style.js
+++ b/node_modules/react-native-imaged-card-view/lib/src/ImagedCardView.style.js
@@ -6,6 +6,15 @@ export default {
     height: 160,
     width: width * 0.9,
     alignSelf: "center",
-    backgroundColor: "transparent"
+    backgroundColor: "transparent",
+    shadowColor: "#000",
+        shadowOffset: {
+            width: 0,
+            height: 8,
+        },
+        shadowOpacity: 0.44,
+        shadowRadius: 10.32,
+
+        elevation: 16,
   }
 };
diff --git a/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.js b/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.js
index 83255f2..e4fb88b 100644
--- a/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.js
+++ b/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.js
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Text, View, Dimensions } from "react-native";
-import Androw from "react-native-androw";
+// import Androw from "react-native-androw";
 import StarReview from "react-native-star-review";
 import ImageContainer from "../ImageContainer/ImageContainer";
 import styles, {
@@ -48,7 +48,7 @@ const CardContainer = props => {
           reviewsText=""
           reviews={1952}
           enableParentheses
-          starColor="white"
+          starColor="black"
           reviewTextStyle={styles.reviewTextStyle}
           {...props}
         />
@@ -57,7 +57,7 @@ const CardContainer = props => {
   };
 
   return (
-    <Androw style={container(width, height, borderRadius, backgroundColor)}>
+    <View style={container(width, height, borderRadius, backgroundColor)}>
       <ImageContainer {...props} />
       <View style={styles.contentContainer}>
         <Text style={titleStyle(titleColor)}>{title}</Text>
@@ -85,7 +85,7 @@ const CardContainer = props => {
           </View>
         </View>
       </View>
-    </Androw>
+    </View>
   );
 };
 
@@ -113,20 +113,20 @@ CardContainer.defaultProps = {
   height: 170,
   borderRadius: 24,
   width: width * 0.75,
-  titleColor: "white",
+  titleColor: "black",
   title: "Island Bali",
-  leftSideTitle: "Days",
+  leftSideTitle: "Calories",
   subtitle: "Indonesia",
-  leftSideColor: "white",
-  dividerColor: "#c4c4c4",
+  leftSideColor: "black",
+  dividerColor: "#FF0000",
   leftSideValue: "4 Days",
-  rightSideColor: "white",
+  rightSideColor: "black",
   rightSideValue: "$1500",
-  rightSideTitle: "Price",
-  subtitleColor: "#dbdbdb",
-  backgroundColor: "#0a96ea",
-  leftSideValueColor: "white",
-  rightSideValueColor: "white"
+  rightSideTitle: "Time",
+  subtitleColor: "#000000",
+  backgroundColor: "#FFFFFF",
+  leftSideValueColor: "black",
+  rightSideValueColor: "black"
 };
 
 export default CardContainer;
diff --git a/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.style.js b/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.style.js
index 90b46b5..2736565 100644
--- a/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.style.js
+++ b/node_modules/react-native-imaged-card-view/lib/src/components/CardContainer/CardContainer.style.js
@@ -11,8 +11,11 @@ export function container(width, height, borderRadius, backgroundColor) {
       width: 0,
       height: 3
     },
-    shadowOpacity: 0.4,
-    shadowRadius: 6
+    shadowOpacity: 0.3,
+    shadowRadius: 3,
+     
+
+     elevation: 14,
   };
 }
 
diff --git a/node_modules/react-native-imaged-card-view/lib/src/components/ImageContainer/ImageContainer.js b/node_modules/react-native-imaged-card-view/lib/src/components/ImageContainer/ImageContainer.js
index f812a38..352e5d6 100644
--- a/node_modules/react-native-imaged-card-view/lib/src/components/ImageContainer/ImageContainer.js
+++ b/node_modules/react-native-imaged-card-view/lib/src/components/ImageContainer/ImageContainer.js
@@ -1,6 +1,6 @@
 import React from "react";
-import { Image } from "react-native";
-import Androw from "react-native-androw";
+import { Image,View } from "react-native";
+// import Androw from "react-native-androw";
 import styles from "./ImageContainer.style";
 
 const defaultImage =
@@ -9,9 +9,9 @@ const defaultImage =
 const ImageContainer = props => {
   const { shadowStyle, source } = props;
   return (
-    <Androw style={shadowStyle}>
+    <View style={shadowStyle}>
       <Image borderRadius={24} style={styles.imageStyle} source={source} />
-    </Androw>
+    </View>
   );
 };
 
